client/src/quiz/components/AnswerOption.js

```diff
@@ -2,30 +2,48 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 function AnswerOption(props) {
-    return (
-        <li className="answerOption">
-            <input
-                type="checkbox"
-                className="checkboxCustomButton"
-                name="checkboxGroup"
-                //checked={props.answerType === props.answer}
-                id={props.answerType}
-                value={props.answerType}
-                //disabled={props.answer}
-                onChange={props.onAnswerSelected}
-            />
-            <label className="checkboxCustomLabel" htmlFor={props.answerType}>
-                {props.answerContent}
-            </label>
-        </li>
-    );
+    if(props.multiChoice == true){
```

Comment:
Why don't you put this around just the Label component instead of the whole return?